var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -52,14 +53,18 @@ class Relion(CMakePackage, CudaPackage):
     depends_on('fltk', when='+gui')
     # cuda 9 not yet supported
     #  https://github.com/3dem/relion/issues/296
-    depends_on('cuda@8.0:8.99', when='+cuda')
+    # depends_on('cuda@8.0:8.99', when='+cuda')
+    depends_on('cuda@9:', when='version==beta-3')
```

Comment:
So version `beta-3` requires `cuda` regardless of whether or not the `+cuda` variant is enabled and no other version requires `cuda` even when `+cuda`?